lib/ansible/module_utils/docker_common.py

```diff
@@ -182,18 +183,33 @@ def __init__(self, argument_spec=None, supports_check_mode=False, mutually_exclu
             required_together=required_together_params,
             required_if=required_if)
 
+        NEEDS_DOCKER_PY2 = (LooseVersion(min_docker_version) < LooseVersion('2.0'))
```

Comment:
Shouldn't `NEEDS_DOCKER_PY2` be `True` if the required version is `>=2.0` instead than `<2.0`?

```suggestion
        NEEDS_DOCKER_PY2 = (LooseVersion(min_docker_version) >= LooseVersion('2.0'))
```

Reply:
Yes, you are right! Thanks for spotting this. I'll create a new PR for fixing this (since this was already merged).